app/components/badge/index.tsx

```diff
@@ -60,9 +60,15 @@ export default function Badge({
         return null;
     }
 
-    // @ts-expect-error: backgroundColor definitely exists
-    const {backgroundColor = rest.backgroundColor || theme.mentionBg, ...restStyle} =
+    const {
+
+        // @ts-expect-error: backgroundColor & borderColor definitely exist
```

Comment:
Should we be asserting here instead of expecting an error? 0/5.

Reply:
All we care about are two of the TextStyle props: backgroundColor & borderColor so that we can pass into the the Animated.Text component.  I think it is safe to cast the flatten() as TextStyle.